### PR TITLE
Add email when volunteer deselected

### PR DESCRIPTION
**Pull request by brandtheisey**
This sends an email to a volunteer if they are deselected / deleted.

Uses text provided by @ultrasaurus .

Addresses #709

### CODE REVIEW

File: assets/js/backbone/apps/tasks/show/controllers/task_show_controller.js [hunk 1]
```diff
@@ -427,6 +427,11 @@ var TaskShowController = BaseView.extend({
       $.ajax({
         url: '/api/volunteer/' + vId,
         type: 'DELETE',
+        data: {
+          taskId: this.model.attributes,
+          vId: vId,
+          steve: "thingy"
```
Comment (csarsam) [inline assets/js/backbone/apps/tasks/show/controllers/task_show_controller.js]: @brandtheisey :grey_question: I am confused about what this field is, could you explain or add a comment?

**Comment by brandtheisey** [comment]
@csarsam data.steve was a debug line that should have been removed.

**Comment by csarsam** [comment]
@brandtheisey sweet, thanks! looks good :+1:

File: assets/email/templates/taskVolunteerRemoved/html.ejs [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<p>Hi <%= metadata.recipient.name %>,</p>
+
+<p>Thanks again for your interest in <%= metadata.task.title %>. We wanted to let you know that someone else was selected for this Opportunity, but plenty of others are still available!</p>
+
+<p>To see a selection of currently available Opportunities, please <a href='<%= metadata.taskLink %>click here</a>.</p>
```
Comment (dhcole) [inline assets/email/templates/taskVolunteerRemoved/html.ejs]: Missing closing ' in `please <a href='<%= metadata.taskLink %>` causes the string to terminate early.

**Comment by dhcole** [comment]
Thanks for scaling the brier patch of notifications. Just that one missing single quote mentioned about and it's good to go.

screen shot 2015-05-01 at 10 46 15 am

**Comment by brandtheisey** [comment]
Closed in favor of #761

**Pull request closed without merging**